index.ts

```diff
@@ -170,7 +185,7 @@ async function main() {
   });
   console.log(safeOpHash);
 
-  const signature = await getNearSignature(safeOpHash);
+  const signature = await getNearSignature(nearAdapter, safeOpHash);
```

Comment:
Oops! Forgot about this detail in the initial version:

```suggestion
  const signature = ethers.solidityPacked(
    ["uint48", "uint48", "bytes"],
    [0, 0, await getNearSignature(nearAdapter, safeOpHash)],
  );
```

Reply:
Oh! Maybe that will help!